src/modules/wohnen/components/OfferDetail.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+
+import React from 'react'
+import { Linking, ScrollView } from 'react-native'
+import { WohnenOfferModel, WohnenFormData, type AccommodationType } from '@integreat-app/integreat-api-client'
+import styled from 'styled-components'
+import type { TFunction } from 'react-i18next'
+import { translate } from 'react-i18next'
+import Caption from '../../../modules/common/components/Caption'
+
+export const formatPrice = (price: number): string => {
+  return (price % 1 === 0) ? `${price}` : `${price.toFixed(2)}`
+}
+
+const formatMonthlyPrice = (price: number): string => (price === 0) ? 'Keine' : `${formatPrice(price)} € monatlich`
+
+type PropsType = {|
+  offer: WohnenOfferModel,
+  t: TFunction
+|}
+
+const Header = styled.Text`
+  padding: 15px 5px 5px;
+  font-weight: 700;
+`
+const MarginalizedView = styled.View`
+  margin: 0 10px;
+`
+
+const RowTitle = styled.Text`
+  flex: 1;
+`
+
+const RowValue = styled.Text`
+  flex: 1;
+`
+
+const Row = styled.View`
+  display: flex;
+  margin: 10px 0;
+`
+
+const ListElement = styled.View`
+  border-bottom-width: 2px;
+  border-bottom-color: ${props => props.theme.colors.themeColor};
+  margin-bottom: 10px;
+`
+
+class OfferDetail extends React.PureComponent<PropsType> {
+  translate (type: 'runningServices' | 'additionalServices' | 'rooms', keys: Array<string>): Array<string> {
```

Comment:
Why do we need to translate here? At least in the WebApp we agreed to only provide the Wohnraumbörse Extra in German. This looks quite strange anyway.

Reply:
I think we use the keys in the webapp aswell. We only translate the keys to German though.